Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -3114,17 +3127,42 @@ def _inject_eval(self, node, func_name):
                     node.pos, self.current_scope_node(), lenv))
         return node
 
+    def _check_inside_class(self, def_node):
+        return isinstance(def_node, Nodes.DefNode) and any(isinstance(node, Nodes.ClassDefNode) for node, _ in self.env_stack)
+
+    def _get_current_class_and_scope(self):
+        return next(
+            (node, scope)
+            for node, scope
+            in self.env_stack[::-1]
```

Comment:
IMHO, `for` and `in` always belong into the same line, especially if an `if` follows, with which `in` is easily confused visually.